functions/dialect.go

```diff
@@ -145,6 +151,12 @@ func (d *dialectImpl) Load(reader io.Reader) error {
 	if err != nil {
 		return err
 	}
+	for _, ti := range d.file.SupportedTypes {
+		if err := defaults.Set(ti); err != nil {
```

Comment:
Can't we just run this against dialectFile? Then we can use it everywhere.

Reply:
yes, this helped removing one pointer.